src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstSide;
+    private int secondSide;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstSide, int secondSide, int height, Color color) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.height = height;
+    }
+
+    public void setFirstSide(int firstSide) {
+        this.firstSide = firstSide;
+    }
+
+    public void setSecondSide(int secondSide) {
+        this.secondSide = secondSide;
+    }
+
+    public void setHeight(int height) {
+        this.height = height;
+    }
+
+    public int getFirstSide() {
+        return firstSide;
+    }
+
+    public int getSecondSide() {
+        return secondSide;
+    }
+
+    public int getHeight() {
+        return height;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstSide * height * 0.5 + secondSide * height * 0.5);

```

Comment:
The formula used to calculate the area of an isosceles trapezoid is [incorrect](https://en.wikipedia.org/wiki/Isosceles_trapezoid#Area). The correct formula is ((firstSide + secondSide) / 2) * height.

Reply:
I had problem with result of area calculation so I made some operations to recode this line. It should be equals, but I can bring back the first option.